README.md

```diff
@@ -1,99 +1,355 @@
+- TDI34 files - In person POS Transactions
+- TDI17 files - Cash & Cheque deposits made to the banks
+- TDI34 (DDF) files - Online card transactions (PayBC and ICE pay)
+
+Files are pushed to sftp and transfered to S3 which triggers a lambda to parse the data into the db. This triggers a subsequent lambda which runs the job of searching for discrepencies in the data and updating the "status" of each row item. After this process has completed another lambda is triggered to generate and output a report to another S3 bucket. 
+
+There is also an alerting and notification lambda which notifies users if data files are missing, or if there are errors in the files during parsing. 
+
+## Table of Contents
+
+
+1. [Tech](#tech)
+2. [Setup](#setup)
+    a. [Requirements](#requirements)
+    b. [Environment Variables](#environment-variables)
+    b. [Tools](#tools)
+    c. [Yarn](#yarn)
+    d. [Node](#node)
+3. [Running The Project](#running-the-project) 
+    a. [Docker](#docker)
+    b. [Database](#database)
+    c. [Lambdas](#lambdas)
+4. [Features](#features)
+5. [Tests](#tests) 
+6. [Documentation](#documentation)
+    a. [Developer Documentation](#developer-documentation)
+    b. [Project Documentation](#project-documentation)
+    c. [Database Documentation](#database-documentation)
+7. [Tools and Support](#tools-and-support)
+    a. [Communication](#communication)
+    b. [Development Tools](#development-tools)
+    c. [Secrets and Keys](#secrets-and-keys)
+    
+
+## Tech
+
+- Database: Postgres
+- Framework: NestJS
+- Deployments: Lambda/Simple Notification Service/App Gateway on AWS
+- File Ingestion: SFTP and AWS Transfer Family
+- Report and File Storage: AWS S3
+- Alerting: GC Notify
+
+
+## Setup
+
+#### Development Tools
+
+Run this command to automatically check for the required prereqs or consult the list below:
+```bash
+make check
+```
+
+- docker 
+- docker-compose
+- node v18
+- postgres v14
+- corepack
+- yarn2
+- awscli
+- jq
+- terraform
+- minio
+- nestjs-cli
+- rclone
+
+#### Requirements
+
+To run the project for development, the following is bare requirement:
+- aws configuration for aws-pcc and minio [Example](./.config/credentials.example)
+        
+        [pcc-aws]
+        aws_access_key_id = <KEY>
+        aws_secret_access_key = <SECRET>
+
+        [minio]
+        aws_access_key_id = <KEY>
+        aws_secret_access_key = <SECRET>
+
+- rclone configuration [Example](./.config/rclone.conf.example)
+        
+        [s3]
+        type = s3
+        provider = AWS
+        env_auth = true
+
+        [minio]
+        type = s3
+        provider = Minio
+        env_auth = true
+        access_key_id = pcc
+        secret_access_key = <LOCAL_MINIO_PASSWORD>
+        endpoint = http://localhost:9000
 
-[![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)](Redirect-URL)
+#### Environment Variables
 
-### MVP - Scope Of Project
+- env file configuration:
```

Comment:
I don't think we need to include the full list here, if our example env is up to date. We can link to that and then provide instructions on filling it in.